user-guide/Advanced_Functionality/Swarming/Tutorials/EnableSwarmingTutorial.md

```diff
@@ -29,138 +37,128 @@ To deploy the package:
-   You can also see this if you open the script via the Automation app and press the **Validate** button to check for errors and warnings.
+   ![Alarm ID usage problem](~/user-guide/images/Swarming_Tutorial_Enable_AlarmID_Usage_Problem.png)
+
+   The *Mask Critical Alarms* script will be displayed as one of the problems that have been found. This script takes as input a reference to an element and will mask all critical alarms of this element. It does so by using a *SetAlarmStateMessage* and includes an identifier reference to the alarm it should mask, but this identifier still uses the old style of referring to an alarm, i.e. a **DataMiner ID/Alarm ID** integer combination. This is must be changed to a **tree ID** instead.
+
+1. Open the *Mask Critical Alarms* script in the Automation app, scroll down, and click *Validate* to check for errors and warnings.
+
+   You will see that a warning is detected. If you were to open this script via DIS in Visual Studio, you would also see an obsolete warning.
 
    ![AlarmID Usage Problem Warning](~/user-guide/images/Swarming_Tutorial_Enable_AlarmID_Usage_Problem_Warning.png)
 
-   Update the code in the foreach block (Line 85) as follows:
+1. Update the code in the *foreach* block (line 85) as follows:
 
-    Before:
+   Before:
 
-    ```csharp
-        var maskRequest = new SetAlarmStateMessage();
-        maskRequest.DataMinerID = alarm.DataMinerID;
-        maskRequest.AlarmId = alarm.AlarmID;
-        maskRequest.DesiredStatus = AlarmUserStatus.Mask;
-        maskRequests.Add(maskRequest);
+   ```csharp
+      var maskRequest = new SetAlarmStateMessage();
+      maskRequest.DataMinerID = alarm.DataMinerID;
+      maskRequest.AlarmId = alarm.AlarmID;
+      maskRequest.DesiredStatus = AlarmUserStatus.Mask;
+      maskRequests.Add(maskRequest);
     ```
 
-    After:
+   After:
 
-    ```csharp
-        var maskRequest = new SetAlarmStateMessage(alarm.TreeID, AlarmUserStatus.Mask, "");
-        maskRequests.Add(maskRequest);
-    ```
+   ```csharp
+      var maskRequest = new SetAlarmStateMessage(alarm.TreeID, AlarmUserStatus.Mask, "");
+      maskRequests.Add(maskRequest);
+   ```
 
-   Press the **validate** button to see if all warnings and errors are gone.
+1. Click *Validate* again to see if all warnings and errors are gone.
 
-   ![AlarmID Usage Problem Fix](~/user-guide/images/Swarming_Tutorial_Enable_AlarmID_Usage_Problem_Fix.png)
+   ![Alarm ID usage problem fix](~/user-guide/images/Swarming_Tutorial_Enable_AlarmID_Usage_Problem_Fix.png)
 
-   If any other scripts or protocols appear, this will also need an update.
-   There will always be an overload or constructor you can use instead, which will refer to the **TreeID** or **AlarmTreeID**.
+1. If any other scripts or protocols are shown in the alarm ID usage summary, update these in a similar way to make sure they refer to the tree ID or alarm tree ID and validate them.
```

Comment:
We should mention to test the updated script to verify the changes. I forgot to add this.